pandas/_testing/contexts.py

```diff
@@ -62,7 +62,6 @@ def set_timezone(tz: str) -> Generator[None, None, None]:
     ...
     'EST'
     """
-    import os
     import time
```

Comment:
`time` is in the standard library, can we move this one to the top of the file too whilst we're here?